src/logging.ts

```diff
@@ -6,7 +6,7 @@ export function logSizeStats() {
   const maxLength = Math.max(...stats.map(([filename]) => filename.length))
   stats.forEach(([filename, prettiedSize]) => {
     const padding = ' '.repeat(maxLength - filename.length)
-    const action = filename.endsWith('.d.ts') ? 'Typed' : 'Built'
+    const action = [".d.cts", ".d.mts", '.d.ts'].some(n => filename.endsWith(n)) ? 'Typed' : 'Built'
```

Comment:
Need to use single quotes `'` and we can share these extensions in constant file